packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -0,0 +1,57 @@
+import { createContext, useEffect, useMemo, useState } from "react";
+
+import { AuthProvider, type AuthProviderParams, useAuth as useAuthCore } from "@crossmint/client-sdk-auth-core";
+import { EVMSmartWallet, SmartWalletSDK } from "@crossmint/client-sdk-smart-wallet";
+
+type AuthWalletProviderParams = AuthProviderParams & {
+    embeddedWallets: {
+        type: "evm-smart-wallet";
+        defaultChain: "polygon-amoy" | "base-sepolia";
```

Comment:
This should be of type `Chain` from the smart wallet package rather than just these two! I initially set it that way to speed up building a demo

Reply:
On second thought I actually think this is fine for the initial release